build.gradle

```diff
@@ -138,6 +139,22 @@ configure(javaProjects) { subproject ->
 
 	[compileJava, compileTestJava]*.options*.compilerArgs = ['-Xlint:all,-options,-processing', '-parameters']
 
+	checkstyle {
+		toolVersion = '10.3'
```

Comment:
Now that we are on Gradle 8.5 can we bump this to the latest and see if we can ditch the dependency on 150?

Reply:
Yeah... Sorry for not being very clear with my previous comment about Spring Javaformat and Gradle `8.5`.

I tried all those possible combinations: as you pointed out only this one works 🤷 

Feel free to play with that on your own: perhaps I still have missed something.

Thanks